WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderWebView.java

```diff
@@ -95,6 +95,10 @@ private void init() {
             this.setWebChromeClient(mReaderChromeClient);
             this.setWebViewClient(new ReaderWebViewClient(this));
             this.getSettings().setUserAgentString(WordPress.getUserAgent());
+            this.getSettings().setDefaultFontSize(
```

Comment:
Let's add a comment here explaining why this is being done. Otherwise a future dev may notice it's not doing anything on their device and think it should be deleted.